pilot/lib/pilot/options.rb

```diff
@@ -189,6 +189,12 @@ def self.available_options
                                      env_name: "PILOT_WAIT_FOR_UPLOADED_BUILD",
                                      description: "Use version info from uploaded ipa file to determine what build to use for distribution. If set to false, latest processing or any latest build will be used",
                                      is_string: false,
+                                     default_value: false),
+        FastlaneCore::ConfigItem.new(key: :reject_previously_submitted_build,
```

Comment:
Not a perfect name. Looking for suggestions to improve this option name

Reply:
Can we name this `:reject_build_waiting_for_review`? This seems to be a bit more specific to what its doing 😊